src/services/corsproxy/corsproxy.service.factory.ts

```diff
@@ -63,8 +74,11 @@ export class WizdomCorsProxyServiceFactory implements IWizdomCorsProxyServiceFac
         return window["WizdomCorsProxy"]["contentWindow"];
     }
     private corsproxyFailure() {
-        window["WizdomCorsProxyState"].corsProxyFailed = true;
-        this.frameService.HandleMessage({command: "WizdomCorsProxyFailed"});
+        if(!window["WizdomCorsProxyState"].session) // only set failure, if its not already had an success
+        {
+            window["WizdomCorsProxyState"].corsProxyFailed = true;
+            this.frameService.HandleMessage({command: "WizdomCorsProxyFailed"});
+        }
```

Comment:
This means that if for some reason it fails on recreating the frame after the token is expired it will still think it's working because it has a session ID from the first frame